tests/golos.referral_tests.cpp

```diff
@@ -124,7 +124,11 @@ BOOST_FIXTURE_TEST_CASE(close_referral_tests, golos_referral_tester) try {
     BOOST_TEST_CHECK(v_referrals.size() == 1);
     BOOST_TEST_CHECK(v_referrals.at(0)["referral"].as<name>() == N(sania));
     BOOST_TEST_CHECK(v_referrals.at(0)["referrer"].as<name>() == N(issuer));
-    step(delay_clear_old_ref);
+
+    step(delay_clear_old_ref / 3 + 1);
```

Comment:
Why don't you use `seconds_to_blocks`? https://github.com/GolosChain/golos-smart/blob/master/tests/golos.publication_tests.cpp#L161
And why do you add 1?